staff/pages/recert.yml

```diff
@@ -10,14 +10,19 @@ requests: # Requests defines all the data requests made available in this page c
     type: KnexRaw # The type of request. Lowdefy uses Knex to talk to Postgresql
     connectionId: postgres # This is the ID of the connection. The connection itself is defined in `lowdefy.yml`
     properties:
-      query: select jsonb_object_agg(sf.form_route, sf.form_data) as form_data,
-        s.updated_at,
-        la.name as agency_name
+      query: |
+        select jsonb_object_agg(coalesce(sf.form_route, 'Null'), sf.form_data) as form_data, s.updated_at, la.name as agency_name, d.s3_documents
         from submissions s
         left join submission_forms sf on (s.submission_id = sf.submission_id)
         left join local_agencies la on (s.local_agency_id = la.local_agency_id)
-        group by sf.submission_id, s.updated_at, la.name
+        left join (
+          select submission_id, jsonb_object_agg(original_filename, s3_url) as s3_documents
```

Comment:
Do we .. need or want to use `jsonb_object_agg` here? (They're not json columns)
Or is this to make the lowdefy frontend happy with JSONB 🤔

Reply:
Yah. Non ideal. I needed a way to associate all the files together for the a single submission

Reply:
Perhaps we should change the db schema for documents to be filename + blob

Reply:
We would need to change 138 on upload.tsx:

```
await deleteFileFromS3(existingRecord.s3Key);
to 
await deleteFileFromS3(existingRecord.data.s3Key);
```

maybe s3Key and filename are columns at the top and we have a column for metadata as a jsonb
Let's consider it later